code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -355,6 +355,12 @@
 					result = (result && val)
 				if("or", "||")
 					result = (result || val)
+				if("|=")
+					result = (result |= val)
+				if("~=")
+					result = (result &= ~val)
```

Comment:
its still wrong, it should be just `|` or just `~` so doing these operations on lists don't fuck up majorly.

The code can not at this point know if the expression is used in a where clause, or a set clause, so it MUST not operate by reference and risk changing values in a where clause, something that would be extremely destructive because it's not expected that the contents of a where clause can modify the things, even if they fail the where clause.